acceptance/tests/base/host/host_test.rb

```diff
@@ -316,3 +316,17 @@
     on default, 'true'
   end
 end
+
+step 'Ensure that a long 128+ character string with UTF-8 characters does not break net-ssh' do
```

Comment:
Yeah, I take your word for it - that's all the behavior that I would expect.

👍